metadata.yaml

```diff
@@ -17,7 +17,7 @@ doc_title: "First-Look Analysis and Feedback Functionality Breakout Group Report
 
 # Author names, ordered as a list. Each author name should be formatted as 'First Last'
 authors:
-  - "Patrick Ingraham"
+  - "Keith Bechtol, Gregory Dubois-Felsmann, Angelo Fausti, Patrick Ingraham (chair), Tony Johnson, Robert Lupton, Tiago Ribeiro, Yousuke Utsumi"
```

Comment:
Thanks for catching this... very poor oversight on my part....

Reply:
I realize the pull request was accidentally against the main branch instead of u_pingraham, which is why it looks as if I am modifying the whole document. I will need to make a new pull request (I don't think it is possible to change the pull request after it is created).